style.css

```diff
@@ -511,3 +519,245 @@ input:valid {
 .img-bar {
   justify-content: center;
 }
+
+@media (min-width: 768px) {
+  .container {
+    width: 100%;
+    height: 3074px;
+  }
+
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+  }
+
+  .nav img {
+    display: none;
+  }
+
+  .desk {
+    display: block;
+    margin-top: 0;
+  }
+
+  .desk ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+    width: 97%;
+    height: 72px;
+    margin-top: -10px;
+    margin-bottom: 20px;
+    align-items: center;
+    gap: 10px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .background {
+    width: 97%;
+    height: 344px;
+    background-image: none;
+  }
+
+  .background-container {
+    height: 100vh;
+    width: 100%;
+    background-image: url(./ImageP1/Header-llustration-desktop-1.png);
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    margin-top: -75px;
+    margin-left: 27px;
+    margin-bottom: 122px;
+  }
+
+  .myheader {
+    width: 70%;
+    height: 114px;
+    margin: 0 auto;
+  }
+
+  .break {
+    display: none;
+  }
+
+  p {
+    width: 70%;
+    height: fit-content;
+    font-size: 18px;
+    margin: 24px auto;
+    margin-top: 0;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  .flex-icons ul {
+    width: 200px;
+    height: 40px;
+    margin: 0 auto;
+    margin-bottom: 45px;
+    padding-left: 0;
+  }
+
+  .flex-icons li {
+    padding: 0 auto;
+  }
+
+  .chevron {
+    margin: 0 auto;
+    margin-top: 0;
+    margin-bottom: 495px;
+  }
+
+  .works {
+    margin-top: 0;
+  }
+
+  header .work {
+    width: 447px;
+    height: 48px;
+    margin-left: 497px;
+    margin-right: 496px;
+    margin-bottom: 52px;
+  }
```

Comment:
- [ ] Kindly consider Implement the changes suggested by the previous reviewer as I can see that you have not implemented this change.